app/payment/channel.go

```diff
@@ -83,3 +97,26 @@ func GetBalInfo(ch perun.ChannelAPI) perun.BalInfo {
 	chInfo := ch.GetInfo()
 	return balsFromState(chInfo.Currency, chInfo.State, chInfo.Parts)
 }
+
+// SubPayChUpdates registers a subscription for updates on this channel.
+func SubPayChUpdates(ch perun.ChannelAPI, notifier PayChUpdateNotifier) error {
+	return ch.SubChUpdates(func(notif perun.ChUpdateNotif) {
+		notifier(PayChUpdateNotif{
+			UpdateID:     notif.UpdateID,
+			ProposedBals: balsFromState(notif.Currency, notif.Update.State, notif.Parts),
+			Version:      fmt.Sprintf("%d", notif.Update.State.Version),
+			Final:        notif.Update.State.IsFinal,
+			Expiry:       notif.Expiry,
+		})
+	})
+}
+
+// UnsubPayChUpdates unregisters a subscription for updates on this channel.
```

Comment:
```suggestion
// UnsubPayChUpdates unsubscribe a subscription for updates on this channel.
```

Reply:
Changed to deletes the existing subscription for ...